.testing/tc2/MOM_input

```diff
@@ -193,7 +193,7 @@ MINIMUM_DEPTH = 1.0             !   [m] default = 0.0
                                 ! The minimum depth of the ocean.
 MAXIMUM_DEPTH = 4500.0          !   [m]
                                 ! The maximum depth of the ocean.
-THICKNESS_CONFIG = "uniform"    !
+THICKNESS_CONFIG = "param"    !
```

Comment:
If this change to the tc2 configuration had been made separately from the rest of the PR, both the code PR and the configuration change PRs would have passed the automated TC testing.  Had we done so, it would mean that the new code is not being tested by the TCs yet, but at least we would not be making exceptions to the rules about tests passing the automated testing before they are accepted.

Reply:
I agree with this suggestion.  We should try to come up with some strategies to preserve testing while eliminating regression reports.

Reply:
In the meantime, I think I can undo this change to tc2 if we adopt the altenative approach of making `default=` an array instead of adding `defaults=`.

Reply:
I undid the modification of tc2